mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/text/TextLayout.kt

```diff
@@ -2,30 +2,31 @@ package com.jakewharton.mosaic.text
 
 import de.cketti.codepoints.codePointCount
 
-internal class TextLayout {
-	var value: String = ""
+public abstract class TextLayout<T : CharSequence>(initialValue: T) {
```

Comment:
I can't see why this and the subtypes need to be public.

Reply:
Thanks, fixed it!